### ISSUE TITLE
Sharing Issues

### ISSUE THREAD

**Issue by Doctorimpossible** [marzika/Snapprefs#115]
There seems to be some issues sharing from google photos. Firstly the upload/choose media button above the shutter loads up google photos but it only loads videos, no photos. Also if you close out of it snapchat will only show black on the view finder until you close it. Secondly sharing directly from google photos will cause snapprefs to force close.

**Comment by marzika** [comment]
The button launches a general intent for media picking, it is handled by the OS.

I will look into the black screen issue, honestly I have no idea why that would happen.

The issue is sharing from the cloud is that the permission for these are exclusive for the querying app (Snapprefs) and when Snapchat tries to use it it gets a null picture because of the permission denial. 3 ways are possible to fix it:

1) Make a copy of the media to the sdcard -> you have to delete them manually later
2) Rewriting of the main Sharing code, lot of work.
3) Download the media

**Comment by Doctorimpossible** [comment]
Ok so the button only sharing videos is on my rom end probably? 

Also I tried downloading and then sharing through google photos and that didn't work. I tried sharing through both the main screen and the device folders section (which holds downloaded content) and both didn't work on multiple images. I did get those images to send over es file explorer though so I believe it's just a google photos issue.
